stats/README.md

```diff
@@ -66,3 +73,11 @@ export STATS__DB_URl="postgres://postgres:admin@localhost:5432/stats"
 export STATS__BLOCKSCOUT_DB_URL="postgres://postgres:admin@localhost:5432/blockscout" 
 cargo run --bin stats-server
 ```
+
+### Docker compose
```

Comment:
could you please also update "Build Using docker-compose" on L25. I guess the link is incorrect